indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -212,3 +226,64 @@
     margin-top: 5px;
     margin-right: 10px;
 }
+
+.abstract-question-list {
+    list-style-type: none;
+
+    > li {
+        label {
+            @include border-right($width: 2px);
+            color: $dark-gray;
+            font-weight: normal;
+            font-style: italic;
+
+            padding-right: 0.5em;
+        }
+
+        .value {
+            .number {
+                vertical-align: middle;
+                margin-left: 0.5em;
+                color: $light-black;
+            }
+
+            .rating {
+                .bullet {
+                    @extend %rating-bullet;
+                    &.full {
+                        @extend %rating-bullet-selected;
+                    }
+                }
+            }
+        }
+    }
+}
+
+.rating-widget {
+    user-select: none;
+    display: flex;
+    flex-direction: row-reverse;
+
+    input[type="radio"] {
+        display: none;
+
+        &:checked ~ label.bullet {
+            @extend %rating-bullet-selected;
+        }
+    }
+    .bullet {
+        cursor: pointer;
+        @extend %rating-bullet;
```

Comment:
Don't you think it's better to always put the `@extend` and `@include` at the top of the selector's body?